pages/bureaux/_id.vue

```diff
@@ -5,7 +5,7 @@
 </template>
 
 <script>
-import website from '~/static/website.json';
+import website from '~/static/bureaux.json';
```

Comment:
Rename `website` to `bureaux`.

Consider removing the `bureaux` attribute in `~/static/bureaux.json` and just put the data directly to avoid using `bureaux.bureaux` and just use `bureaux`.

```
{
  "bureaux":[{emoji:"",nom:"",...}]
}
```

To :

```
[{emoji:"",nom:"",...}]
```